src/components/10-atoms/input-file/ui.test.js

```diff
@@ -0,0 +1,17 @@
+import { Selector } from 'testcafe';
+
+const host = process.env.TEST_HOST_STORYBOOK_URL || 'http://localhost:9999';
+
+fixture('Input file - basic functionality').page(
+  `${host}/iframe.html?id=atoms-input-file--input-file-default`
+);
+
+const TAG = 'axa-input-file';
+const CLASS = '.a-input-file';
+
+test('should render input-file', async t => {
+  const $axaElem = await Selector(TAG);
+  await t.expect($axaElem.exists).ok();
+  const $axaElemEl = await $axaElem.find(CLASS);
+  await t.expect($axaElemEl.exists).ok();
+});
```

Comment:
Not sure if we can even write any tests for the file upload, since a lot of stuff is handled by the browser, but if possible, there might be some tests that would make sense, e.g. validating the color of the upload button or whatever. I didn't test the file upload myself, maybe it's so tiny, we don't need more. @LucaMele You know better if we should add some here. :)

Reply:
@raphaellueckl @NoahLiechti yes u cant really test native file-upload. But noah please add some test where u test the different attributes. Set multiple and check if multiple is set for example

Reply:
also ok if you combine more then one attribute in one test as they are tiny

Reply:
Yes I'll write more tests. Thanks for the feedback!